src/web/extension.ts

```diff
@@ -337,7 +339,7 @@ export function activate(context: vscode.ExtensionContext) {
           case "flow":
             // channel.appendLine("flow message");
             let uri = vscode.Uri.parse(message.uriPath);
-            if (!message.uriPath.endsWith("flow.yaml")) {
+            if (!message.uriPath?.endsWith("flow.yaml")) {
```

Comment:
There was an error happening in main that I found and squashed using this! 


This was the error:
```
Uncaught TypeError TypeError: Cannot read properties of undefined (reading 'endsWith')
    at eval (/Users/bmaldonado/Code/NonPanther/windmill-vscode/src/web/extension.ts:340:34)
```